src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -164,10 +167,11 @@ def query_trace_data(trace_id: str, params: Mapping[str, str]) -> Sequence[Seque
 
 # TODO(wmak): remove ignore once the base is typed
```

Comment:
You don't actually need these, mypy will raise an error if you do `type: ignore` on a line that doesn't have an error. So if someone does type the base they'll have to fix this in order to get the typing to pass.

Reply:
woah, that's awesome!